app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,62 @@
+class WikisController < ApplicationController
+  
+  def index
+    @wikis = Wiki.all
+  end
+
+  def show
+    @wiki = Wiki.find(params[:id])
+  end
+
+  def new
+    @wiki = Wiki.new
+  end
+
+  def create
+
+     @wiki = Wiki.new
+     @wiki.title = params[:wiki][:title]
```

Comment:
When working on the next user story, convert this code to use strong params. Happy to review how that works Monday.